docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -265,6 +265,77 @@ While these are out of scope for the present guide,
+   * - Ancestry
+     - Broadly, characteristics of one’s ancestors. In this guide we refer to geographic ancestry, where your ancestors are from, or lived approximately 500 years ago.
+   * - Racism
+     - The belief that race influences factors outside of the few visible characteristics listed above (skin color, hair, eye color, etc.). Racism often has been used as a justification to see some people as superior or inferior in some dimension of intelligence, abilities, or qualities. 
+   * - Scientific Racism
+     - An attempt by scientists to prove racist theories in order to justify discrimination and allow for racist systems of inequality to continue. The science used was blatantly incorrect and has been debunked. These racist theories are especially dangerous as they can appear to be from valid sources, even though they are not valid science. 
+   * - Institutional Racism
+     - A form of systematic discrimination, based on race or ethnic group, in which societal policies or practices create an advantage for some groups and harm for others. Also known as structural, societal, or systemic racism.  
+   * - Ethnic Discrimination
+     - The belief that an ethnic group is inherently superior or inferior in some dimension of intelligence, abilities, or qualities. 
+   * - Unconscious Bias
+     - Any biases that people have but are not consciously aware of. This can include things not related to individuals or groups such as confirmation bias, or being more receptive to information that aligns to their prior beliefs, and to biases towards individuals or groups, such as favoring people from their own race or ethnicity group, even if they don’t knowingly think that people who are different are inferior. 
+   * - Race-essentialism
+     - The belief that racial groups form discrete genetic categories; that individuals of the same racial category are biogenetically similar; and that different races are fundamentally different. 
+   * - Race-based
+     - The use of race as a key marker in research or policy. This is built on a belief in race-essentialism. 
+   * - Race-conscious
+     - A new approach to how research can include race. Race-conscious research attempts to be aware of the historic roots and the pervasive, systemic nature of oppression, in touch with social reality, and open to the importance of race, while understanding that race itself is a social construct without biological significance. Instead, it frames discrepancies seen between races as a result of historical context or racism.
+   * - Social construct
+     - Any idea or concept that doesn’t have its basis in objective reality but is real because society believes it to be so. Some examples might include the value of money (paper is not inherently valuable, it is valuable because society has collectively agreed that it is), what is fashionable currently (skinny or wide-leg jeans are only better because we believe it to be so), and racial groups. 
```

Comment:
```suggestion
     - Any idea or concept that doesn’t have its basis in objective reality but is real because society believes it to be so. Some examples might include what is fashionable currently (skinny or wide-leg jeans are only better because we believe it to be so) and the value of money (paper is not inherently valuable, it is valuable because society has collectively agreed that it is). Though they have tremendous impacts, racial groupings are social constructs.
```

Just an idea how to make the tone of this one less jarring (going from skinny jeans to racism).